### ISSUE TITLE
FreeCAD crashes when selecting color of a solid

### ISSUE THREAD

**Issue by SnVIZQ** [NixOS/nixpkgs#179228]
### Describe the bug
When color of a solid is being set, FreeCAD crashes. 

### Steps To Reproduce
1. Start FreeCAD
2. Create new document (Ctrl+N)
3. Switch to *Part* workspace and create solid, e.g. *Cube*, from toolbar
4. Double click newly created solid in a view and press "Ctrl+D", in properties click on *Shape color* button (usually a grey color is selected)
5. FreeCAD crashes with:

    (FreeCAD:4863): GLib-GIO-ERROR **: 18:08:34.492: Settings schema 'org.gtk.Settings.ColorChooser' is not installed

### Expected behavior
I expect the Color Chooser Dialog to be displayed where I can select custom color.

### Notify maintainers
@viric, @AndersonTorres 

### Metadata

```console
$ nix-shell -p nix-info --run "nix-info -m"
this path will be fetched (0.06 MiB download, 0.30 MiB unpacked):
  /nix/store/9m5k22zdy4s1yh003ma0igqmmrvncww1-bash-interactive-5.1-p16-dev
copying path '/nix/store/9m5k22zdy4s1yh003ma0igqmmrvncww1-bash-interactive-5.1-p16-dev' from 'https://cache.nixos.org'...
 - system: `"x86_64-linux"`
 - host os: `Linux 5.18.6, NixOS, 22.05 (Quokka), 22.05.git.20e362caddeM`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.1`
 - nixpkgs: `/var/nixos/nixpkgs

$ cd /var/nixos/nixpkgs
$ git log -1
commit 9d9bc0e9f1a234eaffd68b035c66be323976bee0 (HEAD -> release-22.05, origin/release-22.05)
Merge: c0c86bc4c73 b761627947f
Author: Lassulus <github@lassul.us>
Date:   Sun Jun 26 18:23:00 2022 +0200

    Merge pull request #179202 from NixOS/backport-179198-to-release-22.05

    [Backport release-22.05] extra-container: 0.8 -> 0.10
```

### Diff that fixes the issue

Maybe not the best solution, but works fine at least for me (i.e. Color Chooser Dialog is shown and FreeCAD doesn't crash):

```console
$ git diff pkgs/applications/graphics/freecad/default.nix
diff --git a/pkgs/applications/graphics/freecad/default.nix b/pkgs/applications/graphics/freecad/default.nix
index c05f3502aec..6c3f6fac303 100644
--- a/pkgs/applications/graphics/freecad/default.nix
+++ b/pkgs/applications/graphics/freecad/default.nix
@@ -42,6 +42,7 @@
 , swig
 , vtk
 , wrapQtAppsHook
+, wrapGAppsHook
 , xercesc
 , zlib
 }:
@@ -64,6 +65,7 @@ mkDerivation rec {
     pyside2-tools
     gfortran
     wrapQtAppsHook
+    wrapGAppsHook
   ];

   buildInputs = [
             
```

**Comment by symphorien** [comment]
this looks reasonable to me, would you mind opening a PR ?

**Comment by SnVIZQ** [comment]
Sure, I've created [PR#179907](https://github.com/NixOS/nixpkgs/pull/179907), based on master branch. Since the freecad/default.nix is the same in master and release-22.05 branches, it would be nice to have "backport release-22.05" label added to the PR. I don't know how to do that ... or maybe I do not have a permission for such action?

**Comment by heilkn** [comment]
I think there is a regression in 25.11. FreeCAD 1.1.0 crashes when opening a color picker. If I apply the fix from #179907 with the following overlay 
```nix
freecad = (
  super.freecad.overrideAttrs (
    final: prev: {
      nativeBuildInputs = prev.nativeBuildInputs ++ [ super.wrapGAppsHook3 ];
    }
  )
);
```
the crash goes away.

**Comment by symphorien** [comment]
looks like it was removed in https://github.com/NixOS/nixpkgs/pull/451074

I expect that a PR adding it back with a comment linking to this issue would be welcome
